### PR TITLE
Update: Create Hello World Example use case from Serverless API use case in Manifest File

### PR DESCRIPTION
**Pull request by jonife**
please note that "useCaseName" field is now required field when updating a new app template in the  manifest.json file.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
       "dependencyManager": "cli-package",
       "appTemplate": "hello-world",
       "packageType": "Zip",
-      "useCaseName": "Serverless API"
+      "useCaseName": "Hello World Example"
```
Comment (mndeveci) [inline manifest.json:9]: Should we make this `useCaseName` required for the new application templates?

**Review by mndeveci — COMMENTED** [review]
LGTM. Can we also verify that `sam init` integration tests are passing with these changes?

**Review by CoshUS — APPROVED** [review]

**Comment by CoshUS** [comment]
LGTM aside from the comment from Mehmet.

File: manifest.json (hunk 1 above)
Comment (mndeveci) [reply manifest.json:9]: @jonife can you add this test into here?
https://github.com/aws/aws-sam-cli-app-templates/blob/master/tests/unit/test_json.py#L21

File: tests/unit/test_json.py [hunk 2]
```diff
@@ -23,4 +23,6 @@ def test_manifest_template_exist(self):
         for runtime, templates in json_body.items():
             for template in templates:
                 directory = template["directory"]
+                use_case_name = template["useCaseName"]
                 self.assertTrue(Path(directory).exists(), f"{directory} does not exist")
+                self.assertTrue(bool(use_case_name != None), "Template property 'useCaseName' was not specified")
```
Comment (mndeveci) [inline tests/unit/test_json.py:28]: You can use `assertIsNotNone` instead of `asserTrue` as well.

**Review by mndeveci — APPROVED** [review]

**Pull request merged**